### PR TITLE
fix #80140 tellerreport.com

### LINKED ISSUE #80140: www.tellerreport.com
### Issue URL (Ads)
 [https://www.tellerreport.com/sports/2020-02-13%25E2%2580%2593-accusations-of-insanity-and-the-threat-of-the-court%25E2%2580%2593how-fourcade-reacted-to-reztsova%25E2%2580%2599s-words-about-%2522medical-care%2522-.SygbvWLQXI.html](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fwww.tellerreport.com%2Fsports%2F2020-02-13%2525E2%252580%252593-accusations-of-insanity-and-the-threat-of-the-court%2525E2%252580%252593how-fourcade-reacted-to-reztsova%2525E2%252580%252599s-words-about-%252522medical-care%252522-.SygbvWLQXI.html)

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
AdGuard product: | AdGuard Browser extension v3.5.34
Browser: | Chrome
Stealth mode options: | Hide your search queries, <br>Remove X-Client-Data header from HTTP requests, <br>Strip URLs from tracking parameters, <br>Self-destructing third-party cookies (2880), <br>Hide your Referrer from third-parties
Filters: | <b>Ad Blocking:</b><br/> AdGuard Base<br/><br/><b>Privacy:</b><br/> AdGuard Tracking Protection<br/><br/><b>Social Widgets:</b><br/> AdGuard Social Media<br/><br/><b>Annoyances:</b><br/> AdGuard Annoyances

### PR DESCRIPTION
**Pull request by wout-codes**
#80140

### CODE REVIEW

File: EnglishFilter/sections/specific.txt [hunk 1]
```diff
@@ -236,6 +236,8 @@ webshort.in,toonytool.com,btcads24.com,fakechatapp.com,birdurls.com,ashort1a.xyz
...
+! ##.ads
+tellerreport.com,hdmp4mania1.ga,userupload.net,javbraze.com,laptrinhx.com,youtube-video.download,pinkvilla.com,fapjournal.com,javengsub.com,youramateurporn.com,indiansexvideos.porn,onle.co,dausel.co,yesnetwork.com,indiatoday.in,vidcloud.icu,mangafox.cc,mangafull.net,appvn.com,hentaihaven.org,h2mp3.com,livetvcafe.me,photagram.net,tapas.io,freefrontend.com,ally.sh,myplaycity.com,cafemovie.me,mercadojobs.com,thepiratebay.org,sgxnifty.org,justmp4.com,cyberscoop.com,stream-mydirtyhobby.biz,filmcomment.com,youngpornvideos.com,eventcinemas.com.au,findyourlucky.com,iwannawatch.to,linuxmint.com,mobafire.com,themepack.me##.ads
```
Comment (Alex-302) [inline EnglishFilter/sections/specific.txt:240]: unnecessary

File: EnglishFilter/sections/specific.txt [hunk 2]
```diff
@@ -261,6 +263,7 @@ feetporno.com,familyporn.tv,freehardcore.com,webvoyeur.com,moviesand.com,momvids
 !
 !
 !
+tellerreport.com##div.p-4
```
Comment (Alex-302) [inline EnglishFilter/sections/specific.txt:266]: tag is not required here. `.p-4` used for ads only

File: EnglishFilter/sections/css_extended.txt [hunk 3]
```diff
@@ -1,6 +1,8 @@
 !
 ! Extended CSS rules
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/80140
+tellerreport.com#?#article.flex-column > div[class]:contains(/^ADS$/):upward(2)
```
Comment (Alex-302) [inline EnglishFilter/sections/css_extended.txt:5]: or `tellerreport.com#?#.align-content-stretch > div.flex-fill:has(> article.flex-column > div[class=""]:contains(/^ADS$/))`

**Review by Alex-302 — CHANGES_REQUESTED** [review]

File: EnglishFilter/sections/css_extended.txt (hunk 3 above)
Comment (wout-codes) [reply EnglishFilter/sections/css_extended.txt:5]: I prefer mine more. You're one is not long lasting.

File: EnglishFilter/sections/specific.txt (hunk 1 above)
Comment (wout-codes) [reply EnglishFilter/sections/specific.txt:240]: Yeah.

**Comment by wout-codes** [comment]
@Alex-302

**Comment by Alex-302** [comment]
ok. Added.

**Comment by wout-codes** [comment]
@Alex-302 Why did you close it?

**Pull request closed without merging**